stdlib/public/core/Date.swift

```diff
@@ -0,0 +1,78 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+@_silgen_name("swift_get_time")
+@usableFromInline
+func _getTime(
```

Comment:
This will need an availability attribute.

```suggestion
@available(SwiftStdlib 9999, *)
@_silgen_name("swift_get_time")
@usableFromInline
func _getTime(
```